_posts/2024-02-28-How-to-contribute-to-WildFly.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "How to contribute to WildFly"
+date:   2024-02-28
+tags:   WildFly contribution
+author: rchakrab
+description: Article about How to contribute to WildFly.
+---
+
+= How to contribute to WildFly
+
+Welcome to the world of WildFly contributions, a place where your skills can make a real impact! If you're eager to dive in but not sure where to start, this guide is here to simplify the process for you. Whether you're new to open-source or just getting acquainted with WildFly, we've got your back.
```

Comment:
Here it is "open source". That's the noun form. On L62 it is correctly "open-source", as when a noun that is used as an adjective is made up of several words they are joined with '-'.